packages/accounts-base/accounts_client.js

```diff
@@ -17,6 +17,9 @@ export class AccountsClient extends AccountsCommon {
     this._loggingIn = false;
     this._loggingInDeps = new Tracker.Dependency;
 
+    this._loggingOut = false;
+    this._loggingOutDeps = new Tracker.Dependency;
+
```

Comment:
Thanks for the feedback @raix! I didn't use a `ReactiveVar` since these changes were modelled after the existing `loggingIn` code. That being said, using a `ReactiveVar` would definitely clean the code up a bit. I'll look into making these changes, and consider updating the code for `loggingIn` as well (and add some new tests for that part of the code since there don't seem to be any). Thanks again for taking a look at this!